RIOT-OS/RIOT
# drivers/{g,h,i,j}*: add Kconfig modules

### Contribution description
This models the following drivers in Kconfig:
- gp2y10xx
- grove_ledbar
- hd44780
- hdc1000
- hih6130
- hmc5883l
- hts221
- ili9341
- ina2xx
- ina3221
- io1_xplained
- isl29020
- isl29125
- itg320x
- jc42
- my9221 (needed by other driver)
- sdcard_spi (needed by other driver)

### Testing procedure
- Verify that the dependencies are correctly modelled
- Verify that all the correspondent applications have been added to the CI tests
- Green CI

### Issues/PRs references
None
